src/test/regress/sql/multi_tenant_isolation.sql

```diff
@@ -9,6 +9,32 @@ SELECT nextval('pg_catalog.pg_dist_placement_placementid_seq') AS last_placement
 \gset
 ALTER SEQUENCE pg_catalog.pg_dist_placement_placementid_seq RESTART 100000;
 
+BEGIN;
+  SET local client_min_messages TO WARNING;
+  DROP SCHEMA IF EXISTS  "Tenant Isolation" CASCADE;
+  DROP TYPE IF EXISTS test_composite_type CASCADE;
+COMMIT;
```

Comment:
Can this be moved to the end of the file instead? Also it seems like the schema is already dropped there at line 614

Reply:
multi_tenant_isolation_nonblocking creates the schema itself too afaict

Reply:
oh this is corect